framework_tvb/tvb/tests/framework/adapters/uploaders/nifti_importer_test.py

```diff
@@ -112,7 +119,7 @@ def test_import_demo_ts(self):
         """
         This method tests import of a NIFTI file.
         """
-        time_series_index = self._import(self.TIMESERIES_NII_FILE, TimeSeriesVolumeIndex)
+        time_series_index = self._import(self.TIMESERIES_NII_FILE, TimeSeriesVolumeIndex, self.connectivity_gid)
```

Comment:
connectivity reference is not necessary in most cases of NII import.
The test was wrongly "corrected" before after a broken code in importer